linkcheck/cache/urlqueue.py

```diff
@@ -132,7 +138,9 @@ def _put(self, url_data):
                 self.cleanup()
             self.queue.append(url_data)
         self.unfinished_tasks += 1
-        cache.add_result(key, None)  # add none value to cache to prevent checking this url multiple times
+        cache.add_result(
+            key, None
+        )  # add none value to cache to prevent checking this url multiple times
```

Comment:
```suggestion
        # add none value to cache to prevent checking this url multiple times
        cache.add_result(key, None)
```

Reply:
Done, and the other two similar instances.